udata_hydra/routes/status.py

```diff
@@ -60,14 +39,31 @@ async def get_crawler_status(request: web.Request) -> web.Response:
     rate_checked = round(stats_catalog["count_checked"] / total * 100, 1)
     rate_checked_fresh = round(count_checked / total * 100, 1)
 
+    async def get_resources_status_counts(request: web.Request) -> dict[str | None, int]:
```

Comment:
Wow I'm so glad there's a type annotation here, I couldn't believe that we'd have `None` as a key, but we do, and it works, and it's what we want! 🤯